hack/ci/deploy-dev.sh

```diff
@@ -45,4 +45,5 @@ echodate "Successfully got secrets for dev from Vault"
 
 echodate "Deploying ${DEPLOY_STACK} stack to dev"
 ./hack/ci/deploy.sh master ${VALUES_FILE}
+./hack/ci/deploy.sh seed ${VALUES_FILE}
```

Comment:
perhaps a stupid question, but why add this here? Didn't we essentially eliminate the need to run the installer against a seed with #9748? I feel like I'm missing something here.

Reply:
True, this is a leftover from when the cluster migration code was living in the kkp installer. It's now in the seed-ctrl-mgr though.